.github/actions/setup/action.yml

```diff
@@ -38,6 +39,10 @@ runs:
         key: cy-${{ hashFiles('**/package-lock.json') }}
     - run: npm ci
       shell: bash
+    - uses: actions/cache@v3
+      with:
+        path: node_modules/.cache/nx
+        key: nx-${{ github.sha }}
```

Comment:
I'm not gonna remove the cache action above yet until I've finished adding caching to all necessary tasks.

This new action is helpful because, now, when we call a command line `npm run prod`, it also builds the parent tasks. By adding the cache, it _should_ know not to rebuild everything.